gcp/README.md

```diff
@@ -104,6 +101,9 @@ Execute the following commands on the linux machine in order to setup gcloud cli
 ```console
 gcloud init
 ```
+### GCP Quotas
+
+If you created your Google cloud account newly, Google Compute Engine enforces quotas on resource usage for a variety of reasons. For example, quotas protect the community of Google Cloud users by preventing unforeseen spikes in usage, Google keep some soft limitations on the resources, you can always make a request to increase your quota limit. If you are planning to deploy k8ssandra cluster on GKE, you will need to make a request to increase your **Compute Engine API (backend services)** quota to `50` for the future use.
```

Comment:
With a running installation can we check what this value should be _per k8ssandra deployment_ and provide a minimum here for a single deployment instead of a static `50`?

Reply:
It looks like this is captured in https://github.com/k8ssandra/k8ssandra-terraform/issues/11.

Resolving.